research/conditional/utils/argparse.py

```diff
@@ -80,6 +80,20 @@ def introduce_parser_arguments(
     parser.add_argument("--train_dataset_path", type=str, default=None)
     parser.add_argument("--validation_dataset_path", type=str, default=None)
 
+    parser.add_argument("--final_eval_seed", type=int, default=1912)
+    parser.add_argument(
+        "--final_eval_dataloader_batch_size",
+        type=int,
+        default=64,
+        help="Batch size for final evaluation dataloader. It cannot be larger than the batch_size per GPU for the biggest model.",
```

Comment:
I'm a little confused by the biggest model. Why does the model size matter here? Do you mean the biggest batch_size per GPU in the experiment grid?